src/net_processing.cpp

```diff
@@ -1604,6 +1604,34 @@ void static ProcessGetBlockData(CNode* pfrom, const CChainParams& chainparams, c
     }
 }
 
+//! Determine whether or not a peer can request a transaction, and return a reference to it if so.
+Optional<CTransactionRef> static FindTxForGetData(CNode* peer, const CInv& inv, std::chrono::microseconds mempool_req, std::chrono::microseconds longlived_mempool_time) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+{
+    // Check if the requested transaction is so recent that we're just
+    // about to announce it to the peer; if so, they certainly shouldn't
+    // know we already have it.
+    {
+        LOCK(peer->m_tx_relay->cs_tx_inventory);
+        if (peer->m_tx_relay->setInventoryTxToSend.count(inv.hash)) return {};
+    }
+
+    // Send stream from relay memory
```

Comment:
not sure what "stream" means here. `mi` will be an iterator to a `uint256, CTransaction` element on the map. I think this is a relic, but might be worth updating (or just removing) if you touch the code again.

Reply:
It's a relic indeed. The relay pool used to store the serialized transaction stream rather than transaction objects. Fixed.